scripts/verify_deployment.sh

```diff
@@ -61,7 +68,6 @@ echo "River metadata uri is $(cast call ${RIVER} "getMetadataURI()(string)")"
 
 echo "River withdrawal credentials are $(cast call ${RIVER} "getWithdrawalCredentials()(bytes32)")"
 
-REDEEM_MANAGER_PROXY_FIREWALL="0x$(cast storage ${REDEEM_MANAGER} 0xb53127684a568b3173ae13b9f8a6016e243e63b6e8ee1178d6a717850b5d6103 | cut -c 27-)"
```

Comment:
Why remove this one ? The less address to feed into the script the better no ?

Reply:
@0xvv The data is not stored in that contract, so we can't access it :(